styles/widgets/material/diagram.material.less

```diff
@@ -12,6 +12,9 @@
             .dx-accordion-item {
                 border-left: none;
                 border-right: none;
+                .dx-diagram-right-panel-begin-group {
+                    padding-top: 24px;
```

Comment:
Should this padding be same for both compact and regular material themes? If it's not, you can use some existing constant.

Reply:
I tried to find something similar but failed. Could you please give me an example?

Reply:
You don't need a constant if the padding should be same for both compact and regular material themes. Should it be same? 
If it should, it's perfect, keep it as is. If it should not, you can add a new variable in the "size-schemes" folder.